test/test.js

```diff
@@ -26,6 +26,10 @@ describe('ci_info', function() {
   describe('#getInfo', function() {
     var bupenv = Object.keys(process.env);
 
+    beforeEach(function(){
+      delete process.env['TRAVIS'];
```

Comment:
On travis environment, TRAVIS env variable is set to true. 
So it can emulate being on another environment

cf https://travis-ci.org/AdrieanKhisbe/javascript-test-reporter/jobs/101014318